executors/cmdexec/brick_test.go

```diff
@@ -52,8 +46,12 @@ func TestSshExecBrickCreate(t *testing.T) {
 					cmd == "mkdir -p /var/lib/heketi/mounts/vg_xvgid/brick_id", cmd)
 
 			case 1:
+				autobackup := "n"
```

Comment:
You do this in a bunch of places. Perhaps create a short helper function?

Reply:
I could do that, does `utils.BoolToYN(b bool)` sound acceptible?

Reply:
I was originally thinking something more specific to the BackupLVM option, but that'd be fine too. Anything to cut down on the boilerplate and reduce the chance of typos.